### ISSUE TITLE
Hyperparameter selection for Finetuning Full setting.

### ISSUE THREAD

**Issue by sh0416** [princeton-nlp/LM-BFF#48]
Hello.
How to select the hyperparameter choice for Finetuning Full?
The paper takes a validation set by sampling from the training set, which is not an option when the training set is all used.

**Comment by gaotianyu1350** [comment]
Hi,

For fine-tuning on the full dataset, we simply followed the [original RoBERTa paper](https://arxiv.org/abs/1907.11692)
